Cargo.toml

```diff
@@ -14,6 +14,10 @@ build = "build.rs"
 hashbrown = "0.15"
 log = "0.4"
 
+[features]
+default = []
+no-panic = []
```

Comment:
Note: This will depend on the `header-value` (#287) once that PR is merged in order to eliminate the UTF-8 panics.

Reply:
@alexsnaps does this sound reasonable to you?